hack/openshift/fetch-tektoncd-catalog-tasks.sh

```diff
@@ -99,28 +102,44 @@ EOF
 
 
 get_tasks() {
-  local dest_dir="$1"; shift
-  local catalog="$1"; shift
-  local catalog_version="$1"; shift
-  local -n tasks=$1; shift
-  local type="$1"
+  local dest_dir="$1"; shift || true
+  local catalog="$1"; shift || true
+  local catalog_version="$1"; shift || true
+  local type="$1"; shift || true
+  local -n tasks="$1"
```

Comment:
> don't we need shift here?

Not using shift because tasks and stepactions are empty in some cases 

> Why do we need nameref (-n), do we use the reference somewhere outside?

Not from outside but we pass from main function